src/test/kotlin/tornadofx/testapps/GotoViewApp.kt

```diff
@@ -10,11 +10,13 @@ class ContainerView : View("ContainerView") {
 
     override val root = borderpane {
         top {
-            button("Switch view").setOnAction {
-                if (center.lookup("#view1") != null)
-                    subView1.replaceWith(subView2, ViewTransition.Slide(0.2.seconds))
-                else
-                    subView2.replaceWith(subView1, ViewTransition.Slide(0.2.seconds, ViewTransition.Direction.RIGHT))
+            button("Switch view") {
```

Comment:
In this case (with `action` being the only call in the block) there is no difference. I'm not so bothered by lines of code, I just think it's cleaner.